inspire_matcher/utils.py

```diff
@@ -49,9 +49,12 @@ def compute_author_match_score(x_authors, y_authors):
         float: matching score of authors.
 
     """
-    if not x_authors or not y_authors:
+    if not any([x_authors, y_authors]):
```

Comment:
this is equivalent to (probably more clear, at least I had to expand):
```suggestion
    if not x_authors and not y_authors:
```